bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+use Zend\Console;
+use Zend\Loader\StandardAutoloader;
```

Comment:
Maybe add this discussion to the https://github.com/zendframework/zend-view/pull/65?